core/database_arango.py

```diff
@@ -22,6 +23,8 @@
 from arango.exceptions import DocumentInsertError, GraphCreateError
 
 from core.config.config import yeti_config
+from core.events.message import EventType, LinkEvent, ObjectEvent, TagEvent
```

Comment:
Maybe change this for `from core.events import message` so that all other variables are "namespaced" and we know where they come from?

Reply:
Fixed in a941260b55cd712a25cf4f26b75bb34fdf9c8265